scripts/build.sh

```diff
@@ -23,7 +23,7 @@ export GO111MODULE=on
 export ldflags="-X main.version=v${version} -X main.commit=${commit} -X main.builtAt=${built_at} -X main.builtBy=${built_by}"
 
 if [ "$STATIC" == "true" ]; then
-   env GOOS=${GOOS} GOARCH=amd64 go build -v -ldflags="${ldflags} -linkmode external -extldflags -static" -o $BINARY
+   source $(go env GOPATH)/src/github.com/harmony-one/harmony/scripts/setup_bls_build_flags.sh && env GOOS=${GOOS} GOARCH=amd64 go build -v -ldflags="${ldflags} -linkmode external -extldflags -static" -o $BINARY
```

Comment:
why don't you just source the file directly, as you need it in both cases anyway.